IOWalletProximity/IOWalletProximity/Proximity.swift

```diff
@@ -358,6 +413,38 @@ public class Proximity: @unchecked Sendable {
     }
     
     
```

Comment:
There are now two implementations of `buildResponseDocument` method, you could add some comments about its logic and why it is different than other...it is useful for developers who use this library

```suggestion
    /**
 * Builds a response document using a SessionTranscript for authentication.
 * This method is used specifically for OID4VP flows.
 *
 * - Parameters:
 *   - request: Dictionary mapping namespaces to element identifiers which want to share
 *   - issuerSigned: The issuer-signed data for the document
 *   - deviceKey: The private key used for signing the device authentication
 *   - sessionTranscript: The session transcript containing the handover data
 *
 * - Returns: A Document object if document creation succeeded, nil otherwise
 */
```